project-base/storefront/components/Pages/Cart/CartList/CartListItem.tsx

```diff
@@ -49,40 +54,65 @@ export const CartListItem: FC<CartListItemProps> = ({ item, listIndex, onItemRem
         >
             <div className="flex flex-1 basis-full pr-8 vl:basis-auto vl:pr-0">
                 <div className="flex w-24 shrink-0" data-testid={TEST_IDENTIFIER + 'image'}>
-                    <ExtendedNextLink href={item.product.slug} type="product" className="relative h-full w-full">
+                    <ExtendedNextLink href={product.slug} type="product" className="relative h-full w-full">
```

Comment:
why dont you use the previously defined `productSlug`?

Reply:
Ah, nice catch. I forgot to include new link also for the image. Replaced with new link. https://github.com/shopsys/shopsys/pull/2827/files#diff-fcb573c705d7ba7692d64f7c355c17b271c8b622d8571f0afa2cf17fbf7e481e